model/client/admin/peer/peer_reservations_add_request.go

```diff
@@ -8,3 +8,7 @@ type PeerReservationAddRequest struct {
 func (*PeerReservationAddRequest) Method() string {
 	return "peer_reservations_add"
 }
+
+func (*PeerReservationAddRequest) Validate() error {
+	return nil
```

Comment:
public_key is a required field. Also should be base58 encoded, so should we do a check on that? Not sure yet how will talk to Josh